### DISCUSSION TITLE
email field randomly not loaded in Edit mode after changing languages

### DISCUSSION THREAD

**Discussion by khakimvinh** [payloadcms/payload#801] [Q&A] [answered]
Hi everyone, 
I found that the email field sometime randomly not loaded when get in Edit mode of User collection.
When I'm at Edit view of a user, changing language make the field blank. Go back out to the list and get back in doesn't help.

https://user-images.githubusercontent.com/24789737/180661975-7bb45378-7785-4ebe-b037-f6f09c7e37ad.mp4

This happens on both demo (https://demo.payloadcms.com/) and my local running repo. I also tried both Chrome and Firefox both behave the same. Any ideas?

**Comment by jmikrut** [comment, marked as answer]
Hey @khakimvinh — I'm going to personally check into this one today.

Thanks for the report!

**Reply by khakimvinh** [reply]
hi @jmikrut , thanks for looking into this, may I ask if any updates? I don't mean to push or anything, just want to know if there's any work around for this issue, as I also found the same behavior on some other fields in the personal profile page...

**Reply by jmikrut** [reply]
Hey @khakimvinh — this is fixed and released in `1.0.10`!
